debian/include/full

```diff
@@ -11,7 +11,7 @@ RUN APT_UPDATE APT_REDIRECT && \
     APT_INSTALL doxygen LIBCPP_DEV clang-${CLANG_VERSION} \
         libeigen3-dev libscotchmetis-dev libscotchparmetis-dev libsuitesparse-dev \
         libmumps-scotch-dev ${CLANG_PYTHON_PACKAGE}-${CLANG_VERSION} lcov curl \
-        ninja-build libopenmpi-dev openmpi-common openmpi-bin libglu1-mesa libqt5x11extras5 \
```

Comment:
If I recall correctly, these are required to avoid errors (or warnings) in matplotlib visualization in Jupyter notebooks.

Reply:
We don't have to worry actually: https://github.com/dune-community/Dockerfiles/pull/105/commits/c82bfa0a2fc618b6955e778732de7a8f63646e9f